js/analyses/countImages.js

```diff
@@ -0,0 +1,41 @@
+/** @module analyses/countImages */
```

Comment:
`countImages` is a very ambiguous name, from the name I expect to get the amount of image tags in the text. Which is clearly not the case. `getImageStatistics` or `matchKeywordInAltTags` might be better.